package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spawn-wrap",
-  "version": "1.3.4",
+  "version": "1.3.5-a",
```

Comment:
no need to bump the version, we can do this when we release the update.

Reply:
Ok